tasks/configuration.yml

```diff
@@ -52,9 +52,40 @@
     state: absent
   loop:
     - /etc/icinga2/conf.d/services.conf
-    - /etc/icinga2/conf.d/templates.conf
     - /etc/icinga2/conf.d/hosts.conf
 
+# Twilio configuration
+- name: install twilio sms scripts
+  template:
+    src: 'templates/etc/icinga2/scripts/{{ item }}.j2'
+    dest: '/etc/icinga2/scripts/{{ item }}'
+    owner: icinga
+    group: icinga
+    mode: 0755
+    seuser: system_u
+    serole: object_r
+    setype: etc_t
+    selevel: s0
+  loop:
+    - twilio-sms-host-notification.sh
+    - twilio-sms-service-notification.sh
+  when: icinga2_master_twilio_sms_enabled|bool == true
+
+- name: install twilio phone script
+  template:
+    src: 'templates/etc/icinga2/scripts/{{ item }}.j2'
+    dest: '/etc/icinga2/scripts/{{ item }}'
+    owner: icinga
+    group: icinga
+    mode: 0755
+    seuser: system_u
+    serole: object_r
+    setype: etc_t
+    selevel: s0
+  loop:
+    - twilio-phone-notification.sh
```

Comment:
why are there `twilio-sms-host-...` and `twilio-sms-service...` but only `twilio-phone-...`?

Reply:
The phone script can only call the `User` without additional information and thus doesn't need to differ between `Service` and `Host`